guide/inheritance-wallet/recovery.md

```diff
@@ -0,0 +1,151 @@
+    - file: inheritance-transaction-sign/transaction-details
+      alt: Screen showing the transaction details.
+      caption: Christina checks the details again and scrolls down to find her signing device.
+    - file: inheritance-transaction-sign/move-funds-sign-christina-initial
+      alt: Screen showing the transaction details.
+      caption: Christina scrolls down to her inheritance key and hits "sign".
+    - file: inheritance-transaction-sign/move-funds-sign-christina-flow
+      alt: Placeholder screen representing the transaction signing flow.
+      caption: The app takes her through the signing flow.
+    - file: inheritance-transaction-sign/move-funds-sign-christina-done
+      alt: Screen showing the transaction details. It shows that one of the inheritance keys has already signed the transaction.
+      caption: The first signature has been provided.
+images_inheritance-transaction-sign-david:
+    - file: inheritance-transaction-sign/app-home-initial-david
+      alt: App home screen showing that a transaction is ready to be signed.
+      caption: The home screen shows that a transaction is ready to be signed.
+    - file: inheritance-transaction-sign/move-funds-sign-david-initial
+      alt: Screen showing the transaction details. It shows that one of the inheritance keys has already signed the transaction.
+      caption: David goes on to sign the transaction as well.
+    - file: inheritance-transaction-sign/move-funds-sign-david-flow
+      alt: Placeholder screen representing the transaction signing flow.
+      caption: The app takes her through the signing flow.
+    - file: inheritance-transaction-sign/move-funds-success
+      alt: Success screen showing that the transaction has been finalized.
+      caption: With David's signature the required threshold has been reached and the transaction is finalized.
+    - file: inheritance-transaction-sign/move-funds-sign-david-done
+      alt: Screen showing the details of the finalized transaction.
+      caption: The transaction detail screen now shows the finalized transaction.
+    - file: inheritance-transaction-sign/app-home-sending
+      alt: App home screen showing a pending transaction.
+      caption: The home page shows the pending transaction.
+---
+
+<!--
+
+Editor's notes
+
+This page covers a multi-key wallet that uses timelocks to provide additional recovery options.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+# Wallet recovery
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+One day, a couple of years later, Christina and David get sad news that their parents, Alice and Bob, were involved in a deadly car accident. While dealing with the inheritance proceedings, Christina and David log in to their shared family vault in their password manager. 
```

Comment:
> One day, a couple of years later

Later? I assume it's referencing the wallet setup. Maybe change it to `a couple of years after the family set up their inheritance wallet`?